apps/docs/src/app/platform/component-docs/platform-forms/platform-input/platform-input-example/platform-input-auto-complete-validation-example.component.html

```diff
@@ -21,13 +15,9 @@
     <div *ngIf="!itemTemplate" class="fd-popover__body fd-popover__body--no-arrow">
```

Comment:
As far as I can see we do not need `@Input() itemTemplate` option at all, please remove it.

Reply:
It has been removed.